src/main/java/objectsBasicsMandatoryPracticalTasks/Author.java

```diff
@@ -0,0 +1,16 @@
+package objectsBasicsMandatoryPracticalTasks;
+
+class Author {
+    private String firstName, lastName;
```

Comment:
It's better to have these variables declared separately to better readability. I.e.
`private String firstName;`
`private String lastName;`